prisma/migrations/20231113135023_version2/migration.sql

```diff
@@ -0,0 +1,4 @@
+-- AlterTable
+ALTER TABLE "users" ADD COLUMN     "phone" TEXT,
+ADD COLUMN     "status" TEXT NOT NULL DEFAULT 'new',
+ALTER COLUMN "email" DROP NOT NULL;
```

Comment:
Mince, j'ai validé mais `email` peut être null.
Idem pour `phone`

Reply:
Yes, phone a été créé sans la clause `NOT NULL` et cette migration drop la clause pour email

Reply:
D'acc, merci pour la précision.